OpenRA.Mods.Cnc/UtilityCommands/ImportTSMapCommand.cs

```diff
@@ -239,9 +239,6 @@ class ImportTSMapCommand : IUtilityCommand
 			{ 0x03, new byte[] { 0x7E } }
 		};
 
-		// Veins and vein holes
-		static readonly int[] ValidVeinNeighbours = { 0x7E, 0xA7, 0xB2 };
```

Comment:
This was added in #19090. Are we sure we aren't losing something important by removing it?

Reply:
@pchote Is this for future use?

Reply:
The relevant bit is  https://github.com/OpenRA/OpenRA/commit/fcc3008b00fe58b3b3111efa3e7526f5b4f85048#diff-1b46bd8993ce1ce1b586eb8f62f370625423f2f13d779e61293e88fba0825527R241-R466  but I don't know what to make of it.

Reply:
It seems only `0x7E` is used, and it is referenced elsewhere. This is dead code.